src/registrar/tests/test_copy_names_from_contacts_to_users.py

```diff
@@ -0,0 +1,98 @@
+from django.test import TestCase
+
+from registrar.models import (
+    User,
+    Contact,
+)
+
+from registrar.management.commands.copy_names_from_contacts_to_users import Command
+
+
+class TestDataUpdates(TestCase):
+    def setUp(self):
+        """We cannot setup the user details because contacts will override the first and last names in its save method
+        so we will initiate the users, setup the contacts and link them, and leave the rest of the setup to the test(s).
+        """
+
+        self.user1 = User.objects.create(username="user1")
+        self.user2 = User.objects.create(username="user2")
+        self.user3 = User.objects.create(username="user3")
+        self.user4 = User.objects.create(username="user4")
+        # The last user created triggers the creation of a contact and attaches itself to it. @Neil wth is going on?
```

Comment:
**comment**: It's signals again. https://github.com/cisagov/manage.get.gov/blob/main/src/registrar/signals.py#L13 should create the Contact for a user every time the new User object is saved.